homeassistant/config.py

```diff
@@ -86,19 +86,22 @@
 """
 
 
-def _valid_customize(value):
-    """Config validator for customize."""
-    if not isinstance(value, dict):
-        raise vol.Invalid('Expected dictionary')
+CUSTOMIZE_SCHEMA_ENTRY = vol.All(
+    cv.has_at_least_one_key('entity_id', 'entity_id_glob'),
+    cv.ordered_dict(cv.match_all, cv.string))
 
-    for key, val in value.items():
-        if not valid_entity_id(key):
-            raise vol.Invalid('Invalid entity ID: {}'.format(key))
 
-        if not isinstance(val, dict):
-            raise vol.Invalid('Value of {} is not a dictionary'.format(key))
+def _dict_to_list(inp: Any) -> List:
+    if not isinstance(inp, dict):
+        return cv.ensure_list(inp)
+    if 'entity_id' in inp:
+        return [inp]  # sigle entry
+    res = []
 
-    return value
+    for key, val in cv.ordered_dict(OrderedDict)(inp).items():
```

Comment:
The parameter expects a validator, not a class. Since instantiating a class is similar to a validator this works without error, but this dict is simply a one key dict (validator per entry, in order) - see [here](https://github.com/home-assistant/home-assistant/pull/5215/files#diff-ca004a6a81dd74754968d78b3b0bef5dL380)